boards/arm/dragino_lsn50/pinmux.c

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2017 BayLibre, SAS
+ * Copyright (c) 2018 Endre Karlson <endre.karlson@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <device.h>
+#include <init.h>
+#include <pinmux.h>
+#include <sys_io.h>
+
+#include "pinmux/stm32/pinmux_stm32.h"
+
+/* pin assignments for Dragino LSN50 board */
+static const struct pin_config pinconf[] = {
+#ifdef CONFIG_UART_STM32_PORT_1
+	{STM32_PIN_PD5, STM32L0_PINMUX_FUNC_PD5_USART2_TX},
```

Comment:
This does not match what you provided in device tree.
No functional impact since pinmux device tree is not yet enabled, but you should make it be coherent.